element-closest.js

```diff
@@ -1,4 +1,7 @@
 (function (ElementProto) {
+  if (!ElementProto) {
+    return;
```

Comment:
A warning here would be nice so that developers don't have to track down why Element.matches and Element.closest raise errors in Node env.
+1 for this pull request as it enables SSR.